src/api/namespaces/neftyblends/handlers/blends.ts

```diff
@@ -117,7 +117,7 @@ export async function getIngredientOwnershipBlendFilter(params: RequestValues, c
                         b.blend_id = i.blend_id
                     JOIN atomicassets_assets a ON 
                         (i.ingredient_type = 'TEMPLATE_INGREDIENT' AND a.template_id = i.template_id) OR
-                        (i.ingredient_type = 'SCHEMA_INGREDIENT' AND a.schema_name = i.schema_name) OR
+                        (i.ingredient_type = 'SCHEMA_INGREDIENT' AND a.schema_name = i.schema_name AND a.collection_name = i.ingredient_collection_name) OR
```

Comment:
I think also attribute ingredients should have collection name

Reply:
oh yeah you're right, I thought that template_id was part of the ingredient